components/layout/main-navigation.js

```diff
@@ -1,20 +1,37 @@
+import Head from 'next/head';
 import Link from 'next/link';
 import Logo from './logo';
 
 function MainNavigation() {
   return (
-    <header className="flex justify-between bg-blue-900 p-6 place-items-center">
+    <header className="flex justify-between bg-white p-6 place-items-center"
+    style={{ 
+        backgroundColor: "#222b64",
+        paddingLeft: "140px",
+        paddingRight: "140px",
+  
+      }}
```

Comment:
Curious why you're using inline style here, instead of adding the equivalent Tailwind classes?